### PR TITLE
Rename BFT to BTF across the bridge modules for consistency

### PR DESCRIPTION
**Pull request by itsyaasir**
## Issue ticket

Issue ticket link:

### Code conventions

- [x] I have performed a self-review of my code
- [x] Every new function is documented
- [x] Object names are auto explicative
- [x] The PR follows the [project conventions](https://infinityswap.atlassian.net/wiki/spaces/CPROD/pages/23330839/Conventions)
- [x] The code follows the [style guidelines of the project](https://infinityswap.atlassian.net/wiki/spaces/CPROD/pages/24444929/Rust+Conventions)
- [x] The code follows the [project security best practices](https://infinityswap.atlassian.net/wiki/spaces/CPROD/pages/35225620/Security+Considerations)

### Security 

- [x] The PR does not break APIs backward compatibility
- [x] The PR does not break the stable storage backward compatibility

### Testing 

- [x] Every function is properly unit tested
- [x] I have added integration tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] IC endpoints are always tested through the `canister_call!` macro

### CODE REVIEW

File: solidity/src/test_contracts/NewBftBridge.sol [hunk 1]
```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.7;
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
```
Comment (veeso) [inline solidity/src/test_contracts/NewBftBridge.sol:4]: should this file be renamed too?

File: solidity/test/contract_tests/BftBridge.t.sol [hunk 2]
```diff
@@ -4,13 +4,13 @@ pragma solidity ^0.8.7;
 import "forge-std/Test.sol";
```
Comment (veeso) [inline solidity/test/contract_tests/BftBridge.t.sol:4]: should this file be renamed too?

**Review by veeso — CHANGES_REQUESTED** [review]

File: solidity/script/UpgradeBFT.s.sol
Comment (Maximkaaa) [inline solidity/script/UpgradeBFT.s.sol:1]: Rename the file

**Review by Maximkaaa — CHANGES_REQUESTED** [review]
I think these should be renamed also:
image

**Comment by Maximkaaa** [comment]
Yeah, those files from my screenshot got in there after you did search and replace. Sorry :-)

**Review by F3kilo — APPROVED** [review]

**Review by ufoscout — APPROVED** [review]

**Review by Maximkaaa — APPROVED** [review]

**Review by veeso — APPROVED** [review]

**Pull request merged**